src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -32,8 +32,8 @@ azure-mgmt-compute==27.1.0
 azure-mgmt-consumption==2.0.0
 azure-mgmt-containerinstance==9.1.0
 azure-mgmt-containerregistry==10.0.0
-azure-mgmt-containerservice==20.2.0
-azure-mgmt-core==1.3.1
+azure-mgmt-containerservice==20.3.0
+azure-mgmt-core==1.3.2
```

Comment:
> azure-mgmt-core==1.3.2
> 

Since the version of core common package has been upgraded, we recommend merging this PR at the beginning of the next sprint, so that we have more time to test and ensure the stability of CLI